lnbits/core/services.py

```diff
@@ -656,6 +656,7 @@ async def init_admin_settings(super_user: Optional[str] = None) -> SuperSettings
     if super_user:
         account = await get_account(super_user)
     if not account:
+        settings.first_install = True
```

Comment:
This logic is executed only if `lnbits_admin_ui` is true. Is that expected?

Reply:
This only works for the **first** server startup.
**Scenario**:
 - start the server
 - open the browser -> the `first_install` page is served
 - do not enter new username/pass for `super_admin`
 - restart the server
 - open the browser -> `Page not available!` is shown (NOK)
 - expected:  the `first_install` page is served even after a server restart

Reply:
Fixed in this commit: [c903d1d92adc076779acfff3865c1ce138046116](https://github.com/lnbits/lnbits/pull/1977/commits/c903d1d92adc076779acfff3865c1ce138046116)